rmc/utils/generic.py

```diff
@@ -237,6 +276,41 @@ def filter_context_using_gnomad(
     return context_ht
 
 
+def get_annotations_from_context_ht_vep(
+    ht: hl.Table, annotations: List[str] = ["polyphen", "sift"]
```

Comment:
while we're updating this function, should we remove `sift` here since we no longer compare MPC to sift?

Reply:
After discussion, we agreed to keep `sift`

Reply:
Also this made me realize that the default for `annotations` isn't noted in this function's docstring (pushing a fix)